control_node/control_node/calibration.py

```diff
@@ -0,0 +1,182 @@
+import numpy as np
+import time
+from scipy.optimize import minimize
+
+
+class Calibration:
+    def __init__(self, laser, camera, laser_color, logger=None):
+        self.laser = laser
```

Comment:
nit: I'd probably name these laser_client or l_client and cam_client, just to make sure developers understand it is going through the whole ros node architecture.

Reply:
good call, will do